contributors/guide/github-workflow.md

```diff
@@ -74,6 +74,22 @@ git checkout -b myfeature
 Then edit code on the `myfeature` branch.
 
 #### Build
+The following section is a quick start on how to build Kubernetes locally, for more detailed information you can see [kubernetes/build](https://github.com/kubernetes/kubernetes/tree/master/build).
```

Comment:
Can you link the README file instead of dir and use the short link form: 
 https://git.k8s.io/kubernetes/build/README.md

Reply:
Also thanks for squashing the prior set into a cleaner commit.